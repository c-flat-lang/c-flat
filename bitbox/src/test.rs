// use super::*;
//
// #[test]
// fn test_passes() {}
