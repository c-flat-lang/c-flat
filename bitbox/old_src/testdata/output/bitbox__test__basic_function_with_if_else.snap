---
source: bitbox/src/test.rs
expression: contents
---
function max() void{
    @cmp u32 : _tmp0_v0_, _lhs_v0_, _rhs_v0_
    @jumpif _tmp0_v0_, %label_branch_then_label_0
    @jump %label_branch_else_label_1
  %label_branch_then_label_0:   @noop
    @ret u32, _lhs_v0_
  %label_branch_else_label_1:   @noop
    @ret u32, _rhs_v0_
}
